patches/chrome-browser-ui-BUILD.gn.patch

```diff
@@ -1,18 +1,24 @@
 diff --git a/chrome/browser/ui/BUILD.gn b/chrome/browser/ui/BUILD.gn
-index 4a31f9315ce898ed714a7efca6a8afca9d4baf76..767d1686fa4448a2bac4f1fbc7187c8a37e9e2e6 100644
+index 4a31f9315ce898ed714a7efca6a8afca9d4baf76..8c6423caf1762f5be989eff480a67a9d4757cfb6 100644
 --- a/chrome/browser/ui/BUILD.gn
 +++ b/chrome/browser/ui/BUILD.gn
-@@ -608,6 +608,9 @@ static_library("ui") {
+@@ -2,6 +2,7 @@
+ # Use of this source code is governed by a BSD-style license that can be
+ # found in the LICENSE file.
+ 
++import("//brave/browser/ui/ui.gni")
```

Comment:
you don't need to import anything for this, just put it in build/config/brave_build.gni

Reply:
There was an import loop in this case.
As discussed on slack, add a target in `chromium/chrome/browser/ui` with required deps and let `brave/browser/ui` depends on that target in
https://github.com/brave/brave-core/pull/6085/commits/fc0d60843960b3a6fbb441219c8997c217babda4